### ISSUE TITLE
Package GeoidHeightsDotNet 1.0.1 is not compatible with monoandroid10.0

### ISSUE THREAD

**Issue by janusw** [MatejFranceskin/GeoidHeightsDotNet#1]
I'm trying to use GeoidHeightsDotNet in a Xamarin cross-platform project. When building the Android app, I get the following error:

```
Package GeoidHeightsDotNet 1.0.1 is not compatible with monoandroid10.0 (MonoAndroid,Version=v10.0).
Package GeoidHeightsDotNet 1.0.1 supports: net (.NETFramework,Version=v0.0)
```

When using GeoidHeightsDotNet in a portable class library (PCL) based on .NET Standard 2.1, I get this warning:

```
Warning NU1701: Package 'GeoidHeightsDotNet 1.0.1' was restored using
'.NETFramework,Version=v4.6.1, .NETFramework,Version=v4.6.2, .NETFramework,Version=v4.7, .NETFramework,Version=v4.7.1, .NETFramework,Version=v4.7.2, .NETFramework,Version=v4.8'
instead of the project target framework '.NETStandard,Version=v2.1'.
This package may not be fully compatible with your project. (NU1701)
```

**Comment by janusw** [comment]
From what I found on the web, this might possibly be due to a bug in older nuget versions. Maybe one could try to re-generate the nuget package with current tools?

Alternatively, converting the project to .NET standard or .NET core might help to improve cross-platform compatibility?

**Comment by janusw** [comment]
> Alternatively, converting the project to .NET standard or .NET core might help to improve cross-platform compatibility?

I tried this approach. Luckily it was not very hard:
https://github.com/janusw/GeoidHeightsDotNet/commits/net_core_port
The best part is that indeed it fixes my compatibility problems.

@MatejFranceskin Would you be interested in integrating these changes in your repository? (If yes, I'll open a pull request.) Apart from improving compatibility and portability, this also simplifies nuget package generation.

**Comment by MatejFranceskin** [comment]
Yes, please create a PR. Thank you very much!

**Comment by janusw** [comment]
This is fixed by PR #2.
